portality/static/doaj/iso_639-2b.xsd.old

```diff
@@ -0,0 +1,2279 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since this is version 1.0 maybe better to name it according to its version rather than `.old`?